src/context.ts

```diff
@@ -55,6 +57,9 @@ export async function getArgs(inputs: Inputs, buildxVersion: string): Promise<Ar
 
 async function getBakeArgs(inputs: Inputs, buildxVersion: string): Promise<Array<string>> {
   const args: Array<string> = ['bake'];
+  if (inputs.remote) {
+    args.push(inputs.remote);
```

Comment:
Do we need special logic for this if no other opts are specified?
i.e. what happens if we have `{inputs.remote} {inputs.targets...}`?